bt3.cpp

```diff
@@ -0,0 +1,13 @@
+#include <iostream>
+#include <math.h>
+using namespace std;
+int main() {
+	int i,j;
+	for (i=1;i<=9;i++){
+		cout<<"bang cuu chuong cua "<<i<<"la:";
+		for (j=1;j<=10;j++){
+		cout<<i<<"X"<<j<<"="<<i*j;
+		}

```

Comment:
Vì em in không xuống dòng cho nên output nhìn đẹp lắm nha, Cập nhật lại bài này nha:
bang cuu chuong cua 1la:1X1=11X2=21X3=31X4=41X5=51X6=61X7=71X8=81X9=91X10=10bang cuu chuong cua 2la:2X1=22X2=42X3=62X4=82X5=102X6=122X7=142X8=162X9=182X10=20bang cuu chuong cua 3la:3X1=33X2=63X3=93X4=123X5=153X6=183X7=213X8=243X9=273X10=30bang cuu chuong cua 4la:4X1=44X2=84X3=124X4=164X5=204X6=244X7=284X8=324X9=364X10=40bang cuu chuong cua 5la:5X1=55X2=105X3=155X4=205X5=255X6=305X7=355X8=405X9=455X10=50bang cuu chuong cua 6la:6X1=66X2=126X3=186X4=246X5=306X6=366X7=426X8=486X9=546X10=60bang cuu chuong cua 7la:7X1=77X2=147X3=217X4=287X5=357X6=427X7=497X8=567X9=637X10=70bang cuu chuong cua 8la:8X1=88X2=168X3=248X4=328X5=408X6=488X7=568X8=648X9=728X10=80bang cuu chuong cua 9la:9X1=99X2=189X3=279X4=369X5=459X6=549X7=639X8=729X9=819X10=90